dream_layer_backend/report_generator.py

```
@@ -0,0 +1,456 @@
+        
+        return config
+    
+    def _get_models_info(self) -> Dict[str, List[str]]:
+        """Get information about available models"""
+        try:
+            # Try to fetch from ComfyUI API
+            response = requests.get("http://127.0.0.1:8188/models/checkpoints", timeout=5)
+            if response.status_code == 200:
+                checkpoints = response.json()
+            else:
+                checkpoints = []
```

Ruff (0.12.2)

Use ternary operator `checkpoints = response.json() if response.status_code == 200 else []` instead of `if`-`else`-block

Replace `if`-`else`-block with `checkpoints = response.json() if response.status_code == 200 else []`

(SIM108)